appveyor.yml

```diff
@@ -0,0 +1,54 @@
+build: false
+shallow_clone: true
+platform: x86
+clone_folder: c:\projects\phpspec
+
+environment:
+    matrix:
+        - PHP_DOWNLOAD_FILE: php-5.6.14-nts-Win32-VC11-x86.zip
+
+matrix:
+    allow_failures:
+        - PHP_DOWNLOAD_FILE: php-5.6.14-nts-Win32-VC11-x86.zip
+
+skip_commits:
+    message: /\[ci skip\]/
+
+cache:
+    - c:\php -> appveyor.yml
+    - '%LOCALAPPDATA%\Composer'
+    - .phpunit -> phpunit
+    - vendor
+
+init:
+    - SET PATH=c:\php;%PATH%
+    - SET COMPOSER_NO_INTERACTION=1
+    - SET PHP=1
+    - SET ANSICON=121x90 (121x90)
+
+install:
+    - IF EXIST c:\php (SET PHP=0) ELSE (mkdir c:\php)
+    - cd c:\php
+    - IF %PHP%==1 (IF %PHP_DOWNLOAD_FILE%==php-7.0-nts-windows-vc14-x86-r2e38918.zip (appveyor DownloadFile http://windows.php.net/downloads/snaps/php-7.0/r2e38918/%PHP_DOWNLOAD_FILE%) ELSE (appveyor DownloadFile http://windows.php.net/downloads/releases/archives/%PHP_DOWNLOAD_FILE%))
```

Comment:
this can be simplified: ``PHP_DOWNLOAD_FILE`` is never equal to ``php-7.0-nts-windows-vc14-x86-r2e38918.zip``